extensions/amp-list/0.1/amp-list.js

```diff
@@ -451,7 +451,6 @@ export class AmpList extends AMP.BaseElement {
       payload: opt_payload};
 
     if (this.renderedItems_ && append) {
-      this.renderItems_.data = this.renderedItems_.concat(data);
```

Comment:
What is the contract between the amp-list client and the JSON server endpoint? Is the server supposed to send down the full list with more data appended each time, or does each "load more" request contain just the next page and get appended to the internal state of the amp-list? Is this documented?

Reply:
It only contains the next page, not the full list.